### ISSUE TITLE
API for checking if a URI is supported

### ISSUE THREAD

**Issue by kxfang** [airbnb/DeepLinkDispatch#131]
I'm dealing with a legacy code base that can't be migrated over to this library all at once and would like to execute the legacy dispatch logic if we get a link that hasn't been migrated over to DeepLinkDispatch yet. Is there any interest in supporting an API that allows the client to query whether or not a URI is supported?

**Comment by felipecsl** [comment]
You can use a helper method like this:

```java
public static boolean isDeepLink(Intent intent) {
  return intent.getBooleanExtra(DeepLink.IS_DEEP_LINK, false);
}
```

**Comment by kxfang** [comment]
To confirm, the snippet you posted is for dispatched Activities to determine whether or not the Intent is coming from a deep link, is that correct? Our use case is a little different. What we want is something like:
```
if (DeepLinkDispatch.supportsUri(someExternalUri)) {
   UrlDispatch.dispatch(someExternalUri);
} else {
   doSomeSpecialHandling();
}
```

**Comment by felipecsl** [comment]
Oh, I see what you're saying. Yeah that one is not supported yet, but should be easy to add support for. I'll try to get it in in the next few days!

**Comment by kxfang** [comment]
Awesome, thank you!

**Comment by felipecsl** [comment]
Closed via #133
